for_lakshmi.md

```diff
@@ -0,0 +1,28 @@
+[!] `st2ctl status` shall not have root privileges: st2ctl status, st2ctl reload are things that a non-admin user may likely need.
```

Comment:
Though I agree, I just read arma's comments on slack that getting status w/o root may not be possible on systems that use ``init`` as service manager. It might be a restriction imposed because we also added debian to the mix. Got to figure this out.